docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-snippet/icon-snippet.component.ts

```diff
@@ -0,0 +1,49 @@
+import { AfterViewInit, ChangeDetectionStrategy, Component, ElementRef, EventEmitter, Input, Output, Renderer2, ViewChild} from '@angular/core';
+import { FocusIndicatorDirective, FocusIndicatorOriginService } from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'uxd-icon-snippet',
+    templateUrl: './icon-snippet.component.html',
+    styleUrls: ['./icon-snippet.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+
```

Comment:
Remove space between decorator and class.

Reply:
Removed space